tests/base/src/test/java/org/wildfly/security/http/impl/AbstractBaseHttpTest.java

```diff
@@ -438,7 +443,7 @@ public boolean forward(String path) {
         }
     }
 
-    protected CallbackHandler getCallbackHandler(String username, String realm, String password) {
+    protected CallbackHandler getCallbackHandler(String username, String realm, String password, String token) {
```

Comment:
Just a minor comment. Since a bunch of other tests are using this method and don't involve tokens, instead of updating those calls to pass in `null`, we could add a method here with the old method signature that just calls the new method with a `null` value for the token, i.e.,

```
protected CallbackHandler getCallbackHandler(String username, String realm, String password) {
    return getCallbackHandler(username, realm, password, null);
}
```

Then the various other test files updated in this PR will no longer need to be updated.